daemon/logger/copier.go

```diff
@@ -44,8 +43,7 @@ func (c *Copier) copySrc(name string, src io.Reader) {
 	reader := bufio.NewReader(src)
 
 	for {
-		line, err := reader.ReadBytes('\n')
-		line = bytes.TrimSuffix(line, []byte{'\n'})
+		line, _, err := reader.ReadLine()
```

Comment:
Seems we have to handle '\n' or '\r\n' and do things like cuting of the line in buffer `p` ourselves manually if using src.Read(p). What is your concerns of using `bufio.NewReader(src)`?